backend/utils/llm.py

```diff
@@ -94,13 +94,15 @@ def get_transcript_structure(transcript: str, started_at: datetime, language_cod
         'format_instructions': parser.get_format_instructions(),
         'language_code': language_code,
         'started_at': started_at.isoformat(),
+        'tz': tz,
     })
+    print(response)
```

Comment:
remove this log mannnn

Reply:
shit guess I forgot to remove after testing